tests/Directory.Packages.props

```diff
@@ -1,23 +1,22 @@
 <Project>
-    
-    <PropertyGroup>
-        <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    </PropertyGroup>
-
-    <ItemGroup>
-        <PackageVersion Include="FluentAssertions" Version="6.12.0" />
-        <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.9.0-preview-23503-02" />
-        <PackageVersion Include="xunit" Version="2.6.2" />
-        <PackageVersion Include="xunit.runner.visualstudio" Version="2.5.4" />
-        <PackageVersion Include="coverlet.collector" Version="6.0.0" />
-        <PackageVersion Include="NetArchTest.Rules" Version="1.3.2" />
-        <PackageVersion Include="Bogus" Version="34.0.2" />
-        <PackageVersion Include="NSubstitute" Version="5.1.0" />
-        <PackageVersion Include="Respawn" Version="6.1.0" />
-        <PackageVersion Include="Testcontainers" Version="3.6.0" />
-        <PackageVersion Include="Testcontainers.SqlEdge" Version="3.6.0" />
-        <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.0" />
-        <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.0" />
-        <PackageVersion Include="Meziantou.Extensions.Logging.Xunit" Version="1.0.7" />
-    </ItemGroup>
-</Project>
+  <PropertyGroup>
```

Comment:
Is there a standard about using 2 spaces for `xml` files?

Reply:
this is how the `.editorconfig` is configured